### ISSUE TITLE
include_error_in_response_body configuration removed in v3.0.0 ?

### ISSUE THREAD

**Issue by brentsnook** [ianheggie/health_check#83]
This appears to have been removed from this version although it is back again in current master:
https://github.com/ianheggie/health_check/blob/0b799ead604f900ed50685e9b2d469cd2befba5b/lib/health_check.rb#L76

This initializer breaks because the method is missing:

```ruby
HealthCheck.setup do |config|
  config.include_error_in_response_body = false
end
```

**Comment by iamonkara** [comment]
Seems like there is no healthy test coverage for health_check :-1:

**Comment by adamrparsons** [comment]
This is still broken in the latest gem, over a year later.

**Comment by shlima** [comment]
@brentsnook @iamonkara @adamrparsons you can check an [alternative](https://github.com/shlima/health_bit)

**Comment by ianheggie** [comment]
Fixed in master, and am releasing a new release after I finish testing the PRs I have merged

**Comment by ianheggie** [comment]
Fixed in version 3.1.0 released just now
